api/v1/installation_types.go

```diff
@@ -105,6 +105,10 @@ type InstallationSpec struct {
 	// +optional
 	ControlPlaneTolerations []v1.Toleration `json:"controlPlaneTolerations,omitempty"`
 
+	// Replicas defines how many replicas of the control plane core components will be deployed. Defaults to 2.
+	// +optional
+	ControlPlaneReplicas *int32 `json:"controlPlaneReplicas,omitempty"`
```

Comment:
@caseydavenport do you think we should have this field be a string for possible future expansion to non-numeric values? Or is there no concern about switching the field type at a later date?
I think we've done that type of change before.

Reply:
I don't think we need to worry about non-numeric values for this.